src/app/components/account/account-list-item-layout.tsx

```diff
@@ -30,12 +30,14 @@ export function AccountListItemLayout(props: AccountListItemLayoutProps) {
 
   return (
     <Flex
-      width="100%"
       key={`account-${account.index}`}
       data-testid={SettingsSelectors.AccountIndex.replace('[index]', `${account.index}`)}
       cursor="pointer"
       position="relative"
       onClick={onSelectAccount}
+      marginLeft="24px"
+      marginRight="24px"
+      marginBottom="24px"
```

Comment:
Would be good here to use namedSpacingUnit `loose` from the UI library? https://github.com/hirosystems/ui/blob/c9efa75863b29b5f30302b38009dded893cf9b0c/packages/ui-theme/src/sizes.ts#L23